### ISSUE TITLE
Name property missing for "Local development" and "Production development. :A11y_Aspire website_Aspire page_DevTools:

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/aspire.dev#219]
### GitHub Tags:
#A11yTCS; #A11yMAS; #A11ySev2; #A11yWCAG; #DesktopWeb; #ChromiumEdge; #Win11; #Developer Community Partners; #SH_AspireWebsite_Web_Jan26; #WCAG4.1.2; #Name, Role, Value; #ES Chat;

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification."**

Try [ES Chat](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2feng.ms%2fdocs%2fcoreai%2fdevdiv%2fone-engineering-system-1es%2f1es-docs%2fes-chat%2findex&data=05%7c02%7cv-pawanksing%40microsoft.com%7c26cff27270c448f48a0e08de3f569146%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c639017838221175207%7cUnknown%7cTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7c0%7c%7c%7c&sdata=ju8Raw6mIhT5G3gWbZSkS0CAN2zkv2ycA5ifSSqoxOw%3D&reserved=0) to learn more about the MAS rule and how to fix the issue. If you need more help, use our [Teams channel](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fteams.microsoft.com%2fl%2fchannel%2f19%253ADeCtYeREyQHw5jYsz9PPc3hpVw48m-tOlTkQ1jEn0eM1%2540thread.tacv2%2fCAI%2520Accessibility%2520Support%2520Channel?groupId%3Df5eb428c-9e6e-4c16-8143-33fe3961a9ae%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=05%7c02%7cv-pawanksing%40microsoft.com%7c26cff27270c448f48a0e08de3f569146%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c639017838221198042%7cUnknown%7cTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7c0%7c%7c%7c&sdata=DCNVsZe8oDZWqm97AI0ee4OoWWecR6zw6qlhTuzP7MI%3D&reserved=0) or [office hours](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2foutlook.office.com%2fbook%2fmicrosoft2779007%2540microsoft.onmicrosoft.com%2fs%2fbLeGBltMW0qI0WRLsTqx2A2?ismsaljsauthenabled&data=05%7c02%7cv-pawanksing%40microsoft.com%7c26cff27270c448f48a0e08de3f569146%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c639017838221214666%7cUnknown%7cTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7c0%7c%7c%7c&sdata=CnCCdDgEmll2oUQUdMjXusAgV8PneqZVYvzgxAfk7Ik%3D&reserved=0).

### Environment Details:
Application Name: Aspire website
URL: https://aspire.dev/
Chromium Edge: Version 145.0.3775.0 (Official build) dev (64-bit)
Edition: Windows 11 Enterprise Insider Preview
Version: 24H2
OS build: 26540.1001

### Repro steps:

1. Hit the URL: https://aspire.dev/.
2. TAB to "Local development" and "Production development".
3. Observe the issue that name property missing for "Local development" and "Production development.

### Actual Result:
For "Local development" and "Production development" name property is not provided. 

### Expected Result:
For "Local development" and "Production development" appropriate name property should be provided.

### User Impact:
Missing names prevent screen reader users from understanding context, causing confusion during navigation and validation of environment details.

### Attachment:

<img width="1366" height="717" alt="Image" src="https://github.com/user-attachments/assets/879ff9f4-9544-4fb1-b301-bc2c711f4fd6" />

**Comment by IEvangelist** [comment]
Addressed by #222
